tests/test.photo.tab.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { test, expect } from "@playwright/test";
+import HomePage from '../pages/home.page';
+import CreateUnitPage from '../pages/create.unit.page';
+import PhotoTab from '../pages/photo.tab';
+import ServicesTab from '../pages/services.tab'
+
+const validEmail: string = process.env.VALID_EMAIL || '';
+const validPassword: string = process.env.VALID_PASSWORD || '';
```

Comment:
It is advised to use CONSTANT_CASE for such variables. See https://ts.dev/style/#syntax for more details